src/dodal/devices/areadetector/plugins/MJPG.py

```diff
@@ -1,14 +1,16 @@
 import threading
+from abc import ABC, abstractmethod
 from pathlib import Path
 
 import requests
 from ophyd import Component, Device, DeviceStatus, EpicsSignal, EpicsSignalRO, Signal
-from PIL import Image
+from PIL import Image, ImageDraw
 
+from dodal.devices.oav.oav_parameters import OAVConfigParams
 from dodal.log import LOGGER
 
 
-class MJPG(Device):
+class MJPG(Device, ABC):
```

Comment:
I know this has been here for ages, but it would be nice to have a docstring for this device. Maybe we should mention devices should have docstrings on the device standards (https://github.com/DiamondLightSource/dodal/wiki/Device-Standards) ?